packages/react-native/src/primitives/Heading/styles.ts

```diff
@@ -0,0 +1,16 @@
+import { StyleSheet } from 'react-native';
+import { HeadingStyles } from './types';
+
+export const styles: HeadingStyles = StyleSheet.create({
+  text: {
+    color: 'black',
+    // lineHeight: 1,
+  },
+  // these values are completely arbitrary for now
```

Comment:
@dbanksdesign Not pressing, but would definitely like to get your thoughts on what these style values should be

Reply:
Let's work through the value mapping once we have a theme in place